### DISCUSSION TITLE
To add .other permission for the /tpsbar command

### DISCUSSION THREAD

**Discussion by carlosmezquita** [PurpurMC/Purpur#959] [__Completed__] [answered]
Today one of my users found out that people can toggle others tpsbar by just running /tpsbar (player). I would suggest adding a feature to allow only /tpsbar for self but not for others. Adding something like `bukkit.command.tpsbar.other` could fix it.

Thank you!

**Comment by lululombard** [comment]
This is an excellent idea, the TPS bar is really powerful and I want to give it access to trusted players but I don't want them to be able to toggle it for others

**Comment by galacticwarrior9** [comment]
I was about to create a new discussion for this; I'm glad this has been brought up. Players toggling `/tpsbar` for each other has become a minor annoyance on my server. For us, simply restricting the command to server administrators is not an ideal solution as players find immense utility in seeing server lag and ping in real time.

**Comment by granny** [comment, marked as answer]
implemented starting with `build #1961` (https://github.com/PurpurMC/Purpur/commit/fa5b45f62e76cda001bea46febc264e4850adeac)
